aql/aql.proto

```diff
@@ -85,19 +94,11 @@ message EditResult {
 }
 
 
-message EdgeBundle {
-  string src = 1;
-  string label = 2;
-  map<string, float>    float_edges = 3;
-  map<string, int64>    int_edges = 4;
-  map<string, string>   str_edges = 5;
-}
-
 message GraphElement {
   string graph = 1;
   Vertex vertex = 2;
   Edge edge = 3;
-  EdgeBundle edge_bundle = 4;
+  Bundle bundle = 4;
```

Comment:
GraphElement is an RPC envelope data structure. It ties the graph ID to the graph element. Technically it should be oneof for vertex/edge/bundle. Its filled out in the service method calls.